apps/protocol-api/src/prisma/schema.prisma

```diff
@@ -458,6 +474,22 @@ model JobRun {
   name          String
 }
 
+////
+// Tips model
+////
+
+model Tip {
```

Comment:
Looks great! Would this include both the user (tippee) and the tipper?
I know we talked about that at one point but wasn't exactly sure where we landed